src/Views/IframeView.php

```diff
@@ -94,32 +295,20 @@ public function setBodyClasses( $classes ) {
 	 */
 	public function render() {
 		ob_start();
-		?>
-		<!DOCTYPE html>
-		<html <?php language_attributes(); ?>>
-			<head>
-				<meta charset="utf-8">
-				<meta name="viewport" content="width=device-width, initial-scale=1">
-				<title><?php echo apply_filters( 'the_title', $this->title ); ?></title>
-				<?php
-				/**
-				 * Fire the action hook in header
-				 */
-				do_action( 'give_embed_head' );
-				?>
-			</head>
-			<body class="<?php echo implode( ' ', $this->bodyClasses ); ?>">
-				<?php
-				echo $this->body;
-
-				/**
-				 * Fire the action hook in footer
-				 */
-				do_action( 'give_embed_footer' );
-				?>
-			</body>
-		</html>
-		<?php
+
+		$this->loadDefaultConfig();
+
+		if ( $this->modal ) {
+			echo $this->getButtonHTML();
```

Comment:
Probably want some escaping here

Reply:
@bradp This class must believe what is passing to this. Validation must happen where we invoke this. Currently, this class used inside `give_form_shortcode` and I think WordPress shortcode API passes escaped data to this function, so no need to escape here.